nixos/modules/services/hardware/pcscd.nix

```diff
@@ -71,7 +78,7 @@ in
       # around it, we force the path to the cfgFile.
       #
       # https://github.com/NixOS/nixpkgs/issues/121088
-      serviceConfig.ExecStart = [ "" "${getBin package}/bin/pcscd -f -x -c ${cfgFile}" ];
+      serviceConfig.ExecStart = [ "" "${lib.getExe package} -f -x -c ${cfgFile} ${lib.concatStringsSep " " cfg.extraArgs}" ];
```

Comment:
It would be better to use `lib.escapeShellArgs` for this.

Reply:
Or `utils.escapeSystemdExecArgs`?

Reply:
That will not work. That is for systems unit names and it also escapes dashes

Reply:
> That is for systems unit names

No, it isn't.  Please check its [doc](https://github.com/NixOS/nixpkgs/blob/f38266ea8d1c176088264a176236e5b0889f7557/nixos/lib/utils.nix#L56-L74).

[`escapeSystemdPath`](https://github.com/NixOS/nixpkgs/blob/f38266ea8d1c176088264a176236e5b0889f7557/nixos/lib/utils.nix#L42-L54) is for escaping unit names.

Reply:
I have decided to use `lib.escapeShellArgs`. Thank you!